index.html

```diff
@@ -15,7 +15,7 @@
   </head>
   <body>
     <main class="body">
-      <header class="top">
+      <header id="#portfolio"  class="top">
```

Comment:
- [ ] Well done dear @BenjaminMatembe your app has a good design, however, there are a few things that you need to fix before, 

Your Design

<img width="378" alt="1" src="https://user-images.githubusercontent.com/77843050/230997597-eea81542-05cd-4716-b482-587416af9156.png">

- The idea behind the logo name is to tell you to put your name or initials to represent it, please remove my logo text and use your customized one.
- The next thing is the alignment of your texts and icons, see the red line I draw, please make sure your things are aligned on the same line.